src/array-utils.ts

```diff
@@ -0,0 +1 @@
+export const arrayInvetir = (aray:string[]) => aray.reverse();
```

Comment:
nombres en inglés
podría sugerir el nombre `reverseArray`
```suggestion
export const reverseArray = (aray:string[]) => aray.reverse();
```